lib/helpers/utils.js

```diff
@@ -72,3 +74,53 @@ exports.readComponentJSON = function readComponentJSON(componentPath) {
   const content = JSON.parse(fs.readFileSync(path.resolve(componentPath, 'component.json')));
   return content;
 };
+
+exports.validateFixture = function validateFixture(componentPath, fixture, actionName) {
+  const component = exports.readComponentJSON(componentPath);
+
+  if (actionName === 'verifyCredentials') return;
+
+  const exists = (schemaPath) => {
+    if (!fs.existsSync(path.resolve(componentPath, schemaPath))) {
+      print.error(`Schema file ${schemaPath} missing`);
+      return false;
+    }
+    return true;
+  };
+
+  let schema = {};
+
+  const actionOrTrigger = component.actions[actionName] ? 'actions' : 'triggers';
+  const action = component[actionOrTrigger][actionName];
+  if (action.dynamicMetadata) {
```

Comment:
Do you validate the `getMetaModel` function at some point?

Reply:
What do you mean? It is validated for existence and signature in `cmp:validate`. Right now, fixtures are only validated against static schemas though. I can open an issue for an enhancement to validate them against the results of getMetaModel in the future.

Reply:
I just wanted to make sure that was being done somewhere.